db/migrate/20230607202616_create_posts.rb

```diff
@@ -1,13 +1,13 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
-      t.string :Title
-      t.text :Text
-      t.integer :CommentsCounter
-      t.integer :LikesCounter
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, foreign_key: true
 
       t.timestamps
     end
-    
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your readme summary more descriptive an detailed to make your readme professional 👍